docs/refarch/boundedContexts/thirdPartyApi/index.md

```diff
@@ -1,22 +1,228 @@
-# {name} BC
+# Third Party API BC
 
-{overview}
+The Third Party API BC has been implemented with the Mojaloop 2.0 Reference Architecture to enable third-party PISP Operators (typically applications) to interact with the platform.
 
 ## Terms
 
-Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
+The following common terms are used in this BC:
 
-| Term | Description |
-|---|---|
-| Term1 | Description1 |
+| Term | Description  |
+| ---- | ------------ |
+| PISP | Payment Initiation Service Provider |
+| DFSP | Digital Financial Service Provider (Bank |
+| User | DFSP/PISP client (as indicated) |
```

Comment:
This is also generally known as a `Party` in Mojaloop.

I assume `User` is specific to 3PApi?

Reply:
@lewisdaly can you provide your thoughts on this?

Reply:
Perhaps it should be labelled "PISP user" TBC for now.  I have asked Lewis for his guidance.  Thanks

Reply:
Yeah I think this is something like the common user between a PISP and DFSP. I think we used the term user and pisp user interchangeably.